x/gamm/keeper/grpc_query_test.go

```diff
@@ -158,24 +158,24 @@ func (suite *KeeperTestSuite) TestQueryPoolAssets() {
 
 	/*
 		{
-			Weight: sdk.NewInt(200),
+			Weight: sdk.NewInt(200 * GuaranteedWeightPrecision),
```

Comment:
Should we change the RPC to give truncated weights? I feel like we shouldn't, and that its the UI's choice for how much precision they want to give

Reply:
Yeah, I think that should be a UI decision